src/components/Dashboard/Layout.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import React, { Component } from 'react';
+import { Card } from 'react-bootstrap';
+export class Layout extends Component {
```

Comment:
I am wondering if we should refactor these to `src/layouts` - so that we don't have a bunch of classes all called Layout. What do you think?

Reply:
Sounds like a good idea, I'll move it to there

Reply:
Moved to layouts